assets/src/components/v2/persistent_wrapper.tsx

```diff
@@ -27,16 +26,14 @@ const PersistentWrapper: ComponentType<Props> = ({
   useEffect(() => {
     if (isFinished) {
       setVisibleData(data);
-      setRenderKey((n) => n + 1);
       setIsFinished(false);
     }
-  }, [lastFetch]);
+  }, [isFinished]);
 
   return (
     <WrappedComponent
       {...visibleData}
       onFinish={handleFinished}
-      key={renderKey}
       lastUpdate={lastFetch}
```

Comment:
Since in the new version this value is not used at all _within_ the wrapper component itself, it feels like it doesn't need to be included here at all. i.o.w. the "wrapped" component can pull in this value and use it directly or pass it to whatever actually needs it, _if_ anything needs it.

Reply:
Nothing else actually needs this value! Now that page updates are handled purely within each hook, it's not necessary. To me, this makes more sense if hooks handle pagination, and this wrapper component is purely for persisting data and is called when pagination is complete

Reply:
Resurfacing this since it doesn't seem to have been changed yet. Or were you suggesting it should stay here?

Reply:
I was suggesting it should be removed, which it has been! I think we're agreeing here if I understand correctly? `renderKey` isn't being used anymore and is fully removed form the `WrappedComponent`

Reply:
The line I'm commenting on is `lastUpdate`, which is the "this value" I was originally referring to. I guess that might not have been super clear. 😅

Reply:
ahh that value isn't being used by the elevator screens anymore, but it is still used by the `PersistentCarousel` objects. Since they both use the same `PersistentWrapper`, I left it in even though it's optional. I don't see a good way to make including the `lastFetch` conditional without breaking it out into a separate Wrapper (which might be the best option)

Reply:
actually testing out another idea here with optional props that would not require two different wrappers